### PR TITLE
New Mpesa Library alpha v0.1.0

### PR DESCRIPTION
**Pull request by manuelgeek**

### CODE REVIEW

File: lib/ex_mpesa/apis/b2c.ex [hunk 1]
```diff
@@ -0,0 +1,103 @@
+defmodule ExMpesa.B2c do
```
Comment (kamalogudah) [inline lib/ex_mpesa/apis/b2c.ex]: We can capitalize this `B2C`

**Review by okothkongo — APPROVED** [review]

File: lib/ex_mpesa/apis/b2c.ex (hunk 1 above)
Comment (lenileiro) [reply lib/ex_mpesa/apis/b2c.ex]: Nice catch

**Comment by kamalogudah** [comment]
@manuelgeek Another comment I had was with regard to the namespacing for instance we have this `lib/ex_mpesa/apis/b2b.ex` but on the module definition is `ExMpesa.B2B` should we then namespace it to something like `lib/ex_mpesa/b2b.ex`

File: lib/ex_mpesa/apis/b2c.ex (hunk 1 above)
Comment (manuelgeek) [reply lib/ex_mpesa/apis/b2c.ex]: Will work on it

**Comment by manuelgeek** [comment]
> @manuelgeek Another comment I had was with regard to the namespacing for instance we have this `lib/ex_mpesa/apis/b2b.ex` but on the module definition is `ExMpesa.B2B` should we then namespace it to something like `lib/ex_mpesa/b2b.ex`

@kamalogudah  was trying to make the APIs be on one place, but i get you on this

**Comment by kamalogudah** [comment]
Since this will solely be mpesa related.we can avoid the apis names pace we can add it in future let's say we have a v2 from mpesa, that is when we can consider

**Comment by manuelgeek** [comment]
@lenileiro @kamalogudah  is it now good to go?

**Review by lenileiro — APPROVED** [review]
Ship it 🚀🚀

**Pull request merged**